yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/exceptions/YotiSDKAppNotInstalledException.java

```diff
@@ -0,0 +1,13 @@
+package com.yoti.mobile.android.sdk.exceptions;
+
+public class YotiSDKAppNotInstalledException extends YotiSDKException {
```

Comment:
If this exception is valid for any `App` I would remove `Yoti` prefix from its name. Currently seems like it is only launched when the app that is not found is the `Yoti App`

Reply:
We kept the prefix `YotiSDK**` to maintain the naming convention that we have used in other expections.  Our SDK button is still  called `YotiSDKButton`. We can update it whenever we try to update the SDK name😊

Reply:
I see!